netadapter/router/route.go

```diff
@@ -14,10 +16,10 @@ const (
 
 var (
 	// ErrTimeout signifies that one of the router functions had a timeout.
-	ErrTimeout = errors.New("timeout expired")
+	ErrTimeout = protocolerrors.New(false, "timeout expired")
 
 	// ErrRouteClosed indicates that a route was closed while reading/writing.
-	ErrRouteClosed = errors.New("route is closed")
+	ErrRouteClosed = protocolerrors.New(false, "route is closed")
```

Comment:
Is it really a protocol error?

Reply:
Will fix in a separate ticket, for now added TODO